packages/ui-components/style/formeditor.css

```diff
@@ -0,0 +1,284 @@
+  display: none;
+}
+
+li.text-danger {
+  color: var(--jp-error-color1);
+  padding: 5px 0 0 9px;
+}
+
+form.rjsf {
+  width: 100%;
+}
+
+fieldset#root > div > fieldset {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+label.control-label {
+  font-size: var(--jp-content-font-size2);
+  font-weight: 600;
+}
+
+fieldset#root {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.form-group.field {
+  padding: 0px 30px 15px 0;
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+  position: relative;
+  height: fit-content;
+}
+
+.elyra-formEditor datalist {
+  display: none;
+}
+
+.elyra-inputTagList {
+  list-style: none;
+}
+
+.elyra-inputTag {
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+button.elyra-inputTag {
+  cursor: pointer;
+  background: none;
+  border: none;
+  color: var(--jp-ui-font-color2);
+  padding: 0;
+  font-size: var(--jp-ui-font-size1);
+}
+
+.field-object > fieldset > legend {
+  font-size: var(--jp-content-font-size3);
+  padding-bottom: 15px;
+  font-weight: 600;
+}
+
+legend#root__noCategory__title {
+  display: none;
+}
+
+.form-group input[readonly] {
+  background: var(--jp-border-color3);
+}
+
+.form-group.field.field-object {
+  width: 100%;
+  padding: 0;
+}
+
+.jp-ArrayOperations button:disabled {
+  cursor: not-allowed;
+}
+
+.elyra-passwordFieldButton {
+  border: none;
+  border-radius: 5px;
+  background: var(--jp-input-active-background);
+  position: absolute;
+  right: 6px;
+  bottom: 7px;
+}
+
+.elyra-passwordField {
+  position: relative;
+  display: flex;
+}
+
+.elyra-passwordFieldButton:hover {
+  background-color: var(--jp-border-color2);
+}
+
+.elyra-passwordFieldButton svg {
+  width: 25px;
+  display: flex;
+  color: var(--jp-content-font-color0);
+}
+
+input.elyra-inputTag {
+  font-size: var(--jp-ui-font-size1);
+  background: none;
+  border: none;
+  color: var(--jp-ui-font-color2);
+  font-size: var(--jp-ui-font-size1);
+  width: 80px;
+  height: 15px;
+}
+
+.form-group .form-control {
+  padding: 8px;
+  border: 1px solid var(--jp-border-color1);
+  border-radius: 4px;
+  font-size: var(--jp-content-font-size2);
+  margin-top: 10px;
+  width: 100%;
+  background-color: var(--jp-input-active-background);
+  color: var(--jp-content-font-color0);
+}
+
+.elyra-editor-tagList {
+  list-style: none;
+  margin-left: -3px;
+  margin-top: 4px;
+}
+
+.elyra-editor-tag {
+  margin-left: 3px;
+  margin-right: 3px;
+  padding: 0 12px;
+  height: 24px;
+}
+
+button.elyra-editor-tag {
+  cursor: pointer;
+  color: var(--jp-ui-font-color2);
+  font-size: var(--jp-ui-font-size1);
+}
+
+button.elyra-editor-tag.applied-tag {
+  color: var(--jp-ui-font-color1);
+}
+
+button.elyra-editor-tag.unapplied-tag {
+  color: var(--jp-ui-font-color2);
+  white-space: nowrap;
+}
+
+.elyra-editor-tag.tag.unapplied-tag input {
+  border: none;
+}
+
+.elyra-formEditor .elyra-metadataEditor-arrayInput li {
+  padding-left: 0;
+  padding-bottom: 0;
+}
+
+.elyra-metadataEditor
```

Comment:
we shouldn't have css specific to metadataEditior in ui-components, so either this should move back, or if it's actually generic to the form editor we should update the css class name (this applies to all the references in this file)